.clang-format

```diff
@@ -3,7 +3,7 @@ BasedOnStyle: Google
 ---
 Language:               Cpp
 Cpp11BracedListStyle:   true
-Standard:               Cpp11
```

Comment:
@songmeixu why do you change the value of `Standard`? I will get below error with value `c++11`
```
YAML:6:25: error: unknown enumerated scalar
Standard:               c++11

```

Reply:
Oh, it's my clangformat suggested, it's due to versions?

Reply:
mine is `9.0.0` installed with `pip`, which is suggested by @csukuangfj and it works well

Reply:
![image](https://user-images.githubusercontent.com/705582/91962305-eab97780-ed3e-11ea-8171-c24b75877968.png)
I check the doc of clang-format, Çpp11 is alias of keyword latest (which may be c++20), which is deprecated, and as its meaning is changed, I vote the c++11.

Reply:
Where the error comes from, may it cause by locally setting?

Reply:
https://releases.llvm.org/9.0.0/tools/clang/docs/ClangFormatStyleOptions.html

Reply:
no, I changed nothing, just pull & rebase the code, error occurs. What's your version of `clang-format`

Reply:
atually we are not using `Lastest`, we are just using `Cpp11`. That is the 2nd line in your screenshots, so I guess we should change back to `Cpp11`.

Reply:
This is from official doc.. https://clang.llvm.org/docs/ClangFormatStyleOptions.html. Obviously, it's deprecated after 9.0, if keep to the old, use higher versions would get the latest standard , "LS_Latest (in configuration: Latest) Parse and format using the latest supported language version. Cpp11 is a deprecated alias for Latest". I guess when 9.0 version, latest == cpp11, but not right now.

Reply:
> atually we are not using `Lastest`, we are just using `Cpp11`. That is the 2nd line in your screenshots, so I guess we should change back to `Cpp11`.

And the second line indicates it is "c++11 in configuration", the Ls_Cpp11 is a key (used to check internally maybe), not value.

Reply:
OK, I will update my clang-format version and try

Reply:
The error has to be caused by sth.. I doubt it's the clang-format version problem. Mine is:
![image](https://user-images.githubusercontent.com/705582/91965720-7b925200-ed43-11ea-93e1-2312ff8df646.png)

Reply:
according to the link @csukuangfj gives, `Cpp11` instead of `c++11` is the valid option in `9.0.0`, so I guess updating version may work.